doc/subsystems/shell/shell.rst

```diff
@@ -31,9 +31,31 @@ The module can be connected to any transport for command input and output.
 At this point, the following transport layers are implemented:
 
 * UART
+* Segger RTT
 
 See the :ref:`shell_api` documentation for more information.
 
+Connecting to Segger RTT via TCP (on macOS, for example)
```

Comment:
Is this issue exclusive to macOS?  If yes, then delete ``, for example``

Reply:
no, it can be used for any OS but Windows brings a "RTT Viewer" which can be used as well but I can't write anything about it because I have a mac